logs/querysample/querysample.go

```diff
@@ -51,38 +50,39 @@ func transformExplainJSONToQuerySample(logLine state.LogLine, explainText string
 		ExplainSource:     pganalyze_collector.QuerySample_AUTO_EXPLAIN_EXPLAIN_SOURCE,
 		ExplainFormat:     pganalyze_collector.QuerySample_JSON_EXPLAIN_FORMAT,
 		ExplainOutputJSON: &explainJSONOutput,
+		Parameters:        findQueryParameters(explainJSONOutput.QueryParameters),
 	}, nil
 }
 
-var autoExplainTextPlanDetailsRegexp = regexp.MustCompile(`^Query Text: (.+)\s+([\s\S]+)`)
-var herokuAutoExplainWithTabRegexp = regexp.MustCompile(`^Query Text: ([^\t]+)\t([\s\S]+)`)
+var autoExplainTextWithQueryParametersRegexp = regexp.MustCompile(`^Query Text: ([\s\S]+)\r?\n\s*Query Parameters: (.+)\r?\n\s*([\s\S]+)`)
```

Comment:
How is `[\s\S]` different from `.`?

Reply:
With golang, `.` doesn't match with a new line. As "Query Parameters" line is known to be one line (even if the param contains new line, it'll be `\n` instead of being an actual new line there), so I'm matching with `(.+)` there instead of doing `([\s\S]+)`, to make sure that it'll match just a Query Parameters line, and not the other lines.
https://pkg.go.dev/regexp/syntax